Modules/Devices/test/Devices.Application.Tests/Tests/Identities/Commands/CancelDeletionProcessAsSupport/HandlerTests.cs

```diff
@@ -90,11 +95,12 @@ public void Cannot_start_when_given_identity_does_not_exist()
         acting.Should().AwaitThrowAsync<NotFoundException, CancelDeletionAsSupportResponse>().Which.Message.Should().Contain("Identity");
     }
 
-    private static Handler CreateHandler(IIdentitiesRepository? identitiesRepository = null, IEventBus? eventBus = null)
+    private static Handler CreateHandler(IIdentitiesRepository? identitiesRepository = null, IEventBus? eventBus = null, IPushNotificationSender? pushNotificationSender = null)
     {
         identitiesRepository ??= A.Fake<IIdentitiesRepository>();
         eventBus ??= A.Fake<IEventBus>();
+        pushNotificationSender ??= A.Fake<IPushNotificationSender>();
```

Comment:
```suggestion
        pushNotificationSender ??= A.Dummy<IPushNotificationSender>();
```
(same for the eventBus and repository above)